multires-lyrics-search/tests/test_multireslyrics.py

```diff
@@ -106,4 +113,15 @@ def test_query(tmpdir, queries_and_expected_replies):
 
             # check the number of docs returned
             # note. the TOP K reflects nr of matches per chunk
-            assert len(matches) <= TOP_K * len(chunks)
+            assert len(matches) <= TOP_K * JINA_SHARDS
+            all_results.append(
+                [
+                    query,
+                    {
+                        "match-level": match_result,
+                        "chunk-level": query_chunk_results
+                    }
+                ]
+            )
+    with open(f'results batch size {BATCH_SIZE}.json', 'w') as f:
```

Comment:
remove this wriing right?

Reply:
I think it's useful for debugging. But I can see that other examples don't have this, so maybe I should remove it to keep things consistent?

Reply:
Yes, please remove it. Once one wants to debug can start doing their prints. But this test is a test to be run by our CI CDs

Reply:
Done